db/schema.rb

```diff
@@ -16,6 +16,25 @@
   # These are extensions that must be enabled in order to support this database
   enable_extension "plpgsql"
 
+  create_table "companies", force: :cascade do |t|
```

Comment:
@benichu can you please do a `rake db:migrate:reset` to update your schema? These tables should not be in the schema.